code/modules/reagents/chemistry/machinery/pandemic.dm

```diff
@@ -216,6 +221,25 @@
 		P.name = "Releasing Virus - [D.name]"
 		printing = null
 
+/obj/machinery/computer/pandemic/proc/print_goal_orders()
+	if(stat & (BROKEN|NOPOWER))
+		return
+
+	playsound(loc, 'sound/goonstation/machines/printer_thermal.ogg', 50, TRUE)
+	var/obj/item/paper/P = new /obj/item/paper(loc)
+	P.name = "paper- 'Viral Samples Request'"
+
+	var/info_text = "<div style='text-align:center;'><img src='ntlogo.png'>"
+	info_text += "<h3>Viral Sample Orders</h3></div><hr>"
+	info_text += "<b>Viral Sample Orders for [station_name()]'s Virologist:</b><br><br>"
+
+	for(var/datum/virology_goal/G in GLOB.virology_goals)
+		info_text += G.get_report()
+		info_text += "<hr>"
+
+	P.info = info_text
```

Comment:
Turn "info_text" into a list then ".Join("")" it all when adding to "P.info", check CONTRIBUTING.md for an example.